packages/webapp/src/components/EditCropVariety/index.jsx

```diff
@@ -101,15 +107,22 @@ export default function PureEditCropVariety({
       />
 
       <div className={styles.cropLabel}>{cropNameLabel}</div>
-      <img
-        src={crop_variety_photo_url}
-        alt={cropVariety.crop_common_name}
-        className={styles.circleImg}
-        onError={(e) => {
-          e.target.onerror = null;
-          e.target.src = 'crop-images/default.jpg';
-        }}
-      />
+
+      {showSpinner ? (
+        <div style={{ height: 'fit-content' }}>
```

Comment:
This is probably not working as you intended...? Nothing changes when removing `fit-content`(or `div`) for me.

Reply:
@kathyavini  You just have to shut down the MinIO and then try to upload a file. It will fail on the saga file, and the spinner will persist instead of any other previous image.

Reply:
So this is so subtle maybe it's ridiculous to have been thinking about it, but if you remove the `<div>`, the Spinner takes up more space than the image, and the rest of the page moves down during loading:

https://user-images.githubusercontent.com/82857964/235545683-cf559c95-94ff-4d8e-8d42-fecbb5c76d31.mov



While with the `<div>` included, the Spinner is smaller than the image so the rest of the page moves up during loading:

https://user-images.githubusercontent.com/82857964/235545697-3c8fdda3-d76d-4914-b30c-79358bed9afa.mov


The only reason I adjusted was because the move-up behaviour was what was seen in the other two views and I wanted to be consistent; although I doubt anyone will notice at all, especially unless comparing them one after another!

Reply:
@SayakaOno Oh wow, that freaks me out a little 😬